stl/inc/format

```diff
@@ -191,6 +212,31 @@ constexpr const _CharT* _Parse_width(const _CharT* _Begin, const _CharT* _End, _
     return _Begin;
 }
 
+template <class _CharT, _Parse_spec_callbacks<_CharT> _Callbacks_type>
+constexpr const _CharT* _Parse_precision(const _CharT* _Begin, const _CharT* _End, _Callbacks_type&& _Callbacks) {
+    ++_Begin;
+    _CharT _Ch = '\0';
+    if (_Begin != _End) {
+        _Ch = *_Begin;
+    }
+    if ('0' <= _Ch && _Ch <= '9') {
+        int _Precision = 0;
+        _Begin         = _Parse_nonnegative_integer(_Begin, _End, _Precision);
+        _Callbacks._On_precision(_Precision);
+    } else if (_Ch == '{') {
+        ++_Begin;
+        if (_Begin != _End) {
+            _Begin = _Parse_arg_id(_Begin, _End, _Precision_adapter<_CharT, _Callbacks_type>(_Callbacks));
+        }
+        if (_Begin == _End || *_Begin != '}') {
+            throw format_error("Invalid format string.");
+        }
+    } else {
+        throw format_error("missing precision specifier");

```

Comment:
Immediately above, `"Invalid format string."` was capitalized with a period, but this has neither. These exceptions should be consistent.